pkg/reader/ccip.go

```diff
@@ -399,22 +399,26 @@ func (r *ccipChainReader) GetExpectedNextSequenceNumber(
 
 func (r *ccipChainReader) NextSeqNum(
 	ctx context.Context, chains []cciptypes.ChainSelector,
-) ([]cciptypes.SeqNum, error) {
+) (map[cciptypes.ChainSelector]cciptypes.SeqNum, error) {
 	cfgs, err := r.getOffRampSourceChainsConfig(ctx, chains)
 	if err != nil {
 		return nil, fmt.Errorf("get source chains config: %w", err)
 	}
 
-	res := make([]cciptypes.SeqNum, 0, len(chains))
+	res := make(map[cciptypes.ChainSelector]cciptypes.SeqNum, len(chains))
 	for _, chain := range chains {
 		cfg, exists := cfgs[chain]
 		if !exists {
-			return nil, fmt.Errorf("source chain config not found for chain %d", chain)
+			r.lggr.Warnw("source chain config not found for chain %d, chain is skipped.", chain)
```

Comment:
This log is malformed, if you want formatted logs should use `Warnf`, `Warnw` is a structure log so should look like

```golang
r.lggr.Warnw("source chain config not found for chain %d, chain is skipped.", chain)
```